AgendaUWP/Service/Model/Contact.cs

```diff
@@ -68,6 +70,49 @@ public override string ToString()
         {
             return FullName + NickName + Age + Phone;
         }
+
+        public void Validate()
+        {
+            var messages = new List<string>();
+
+            if (IsNullOREmpty(_fullname))
```

Comment:
You can use the string.IsNullOrEmpty method instead of creating a new one.

Reply:
Cool! Done!